plugins/ua_network_tcp.h

```diff
@@ -11,12 +11,24 @@ extern "C" {
 #include "ua_server.h"
 #include "ua_client.h"
 
+#ifdef _WIN32
+# include <winsock2.h>
+#else
+# include <sys/time.h>
+# include <sys/types.h>
+# include <unistd.h>
+#endif
```

Comment:
Can you put these includes into the corresponding source file to keep the header free of any non necessary includes?
That avoids that these files will also be included in any user program.

Reply:
It's difficult because we need to define fd_set and timeval for UA_select